contribute/merge-pull-request.md

```diff
@@ -30,74 +30,75 @@ The pull request title should be formatted according to `<Area>: <Summary>` (Bot
-Pull requests targeting `main` should use the `.x` milestone of the next minor (or major) version (you can find that version number inside the `package.json` file).
-Backport pull requests should use the version of the target branch (e.g. `9.4.x` for the `v9.4.x` branch).
+- For every major and minor release, there is a milestone ending with `.x` (for example, `10.0.x` for the 10.0.x releases).
+- Pull requests targeting `main` should use the `.x` milestone of the next minor (or major) version (you can find that version number inside the `package.json` file).
+- Backport pull requests should use the version of the target branch (for example, `9.4.x` for the `v9.4.x` branch).
 
-### Include in changelog and release notes?
+### What to include in changelog and release notes
 
-At Grafana we generate the [changelog](https://github.com/grafana/grafana/blob/main/CHANGELOG.md) and [release notes](https://grafana.com/docs/grafana/latest/release-notes/) based on merged pull requests. Including changes in the changelog/release notes is very important to provide a somewhat complete picture of what changes a Grafana release actually includes.
+At Grafana we generate the [changelog](https://github.com/grafana/grafana/blob/main/CHANGELOG.md) and [release notes](https://grafana.com/docs/grafana/latest/release-notes/) based on merged pull requests. Including changes in the changelog (release notes) is very important to provide a somewhat complete picture of what changes a Grafana release actually includes.
 
-There's a GitHub action available in the repository named [Update changelog](https://github.com/grafana/grafana/blob/main/.github/workflows/update-changelog.yml) that can manually be triggered to re-generate the changelog and release notes for any release.
+There's a GitHub action available in the repository named [Update changelog](https://github.com/grafana/grafana/blob/main/.github/workflows/update-changelog.yml) that can be triggered manually to re-generate the changelog and release notes for any release.
 
 Exactly what changes should be added to the changelog is hard to answer but here's some general guidance:
 
 - Include any bug fix in general.
 - Include any change that you think would be interesting for the community as a whole.
-- Skip larger features divided in multiple pull requests since they might go into the release blog post/What's New article.
-- Use your best judgement and/or ask other maintainers for advice.
-- Including a change in error rather than skipping one that should have been there is better.
-- Always keep [Format the pull request title](#format-the-pull-request-title) in mind.
+- Skip larger features divided into multiple pull requests since they might go into the release blog post (What's New article).
+- Use your best judgement and, if you aren't sure, then ask other maintainers for advice.
+- It's better to include a change in error rather than skip one that should have been there. Err on the side of inclusion.
```

Comment:
this sentence is kind of difficult to parse. perhaps this is better: 
"It's better to err on the side of inclusion. Introducing an unnecessary or duplicate change in the changelog is better than leaving out a change."

Reply:
nice